.github/workflows/ci-build.yaml

```diff
@@ -151,14 +150,101 @@ jobs:
           df -h /
           tar -xzf ../maven-repo.tgz -C ~
           df -h /
-      - name: Build on ${{ matrix.java }}
+      - name: cd extensions-core && mvn test
         run: |
-          ./mvnw -V -ntp ${BRANCH_OPTIONS} \
-            clean verify
-      - name: Fail if there are uncommitted changes
+          cd extensions-core
+          ../mvnw -V -ntp ${BRANCH_OPTIONS} \
+            -Dformatter.skip -Dimpsort.skip -Denforce=false -Dcamel-quarkus.update-extension-doc-page.skip \
```

Comment:
Maybe we could look at doing what Quarkus does and combine most of these under a `quickly` profile. E.g `mvn clean install -Dquickly`.

Reply:
Hm... I am a bit undecided, what should be there in `-Dquickly`. I tend to think that most of times when I want to build quickly, `-Dformatter.skip -Dimpsort.skip -Denforce=false -Dcamel-quarkus.update-extension-doc-page.skip` should not be there, because in that way, I'd end up with unformated source tree with bad deps and stale docs.

Reply:
It depends what you want to achieve i.e. if you check out master and you want a quick way to install everything because you are working on a side project that depends on latest camel-quarkus bits, you don't care about formattings or docs

Reply:
Yeah, that sounds like a valid use case. Would you mind creating a followup issue?

Reply:
https://github.com/apache/camel-quarkus/issues/1607